doc/060_forget.rst

```diff
@@ -215,10 +215,9 @@ single snapshot on 2017-09-30 (Sat) will count as a daily, weekly and monthly.
 Let's explain this with an example: Suppose you have only made a backup
 on each Sunday for 12 weeks. Then ``forget --keep-daily 4`` will keep
 the last four snapshots for the last four Sundays, but remove the rest.
-Only counting the days which have a backup and ignore the ones without
-is a safety feature: it prevents restic from removing many snapshots
-when no new ones are created. If it was implemented otherwise, running
-``forget --keep-daily 4`` on a Friday would remove all snapshots!
+
+Restic has a safety feature, preventing restic from removing many snapshots, when no newer exist.
+If implemented otherwise, running ``forget --keep-daily 4`` on a Friday would remove all snapshots.
```

Comment:
Hm, now the text is technically incorrect. Thank you very much for your contribution, but I think the right way would be to add a complete example (with restic output and so on) instead of changing the text. So I'm inclined to close this PR for now.